### DISCUSSION TITLE
Optional Indexing Of Package Descriptions (*useful* feature) HOPEFULLY

### DISCUSSION THREAD

**Discussion by hothilux-21** [chocolatey/choco#3445] [Ideas] [answered]
Could we choose (when we install choco) for additinal desciption assets for each app when searching each package.

SO if i search something I want there with the package string and ID and version it says a description the community gathers for it, both the GUI and CUI if possible. And also, we can voluntarily commit to developing descriptions for the source database for corresponding descriptions that are brief, and maybe in the description the Author

Example: choco search quickcpu

quickcpu 4.5.1 by (CODERBAG): CPU Unparking and Power Plan Configuration Software [Approved] Downloads cached for licensed users - Possibly broken for FOSS users (due to original download location changes by vendor)

Or like (PACKAGE NAME) (VER) (Author or associated developer) (BRIEF AND USEFUL DESC) and whatever disclaimer goes after etc.

I will try to commit if this suggestion is taken, and I hope this becomes a vital (but optional) feature so we don't have to scour for the package off MajorGeeks to see if the package name corresponds to the TRULY intended program we want the package of from online.

We can all commit to making descriptions that can become indexed into the choco program soon to the best of our ability given this idea is taken onboard.

Thanks all,

StormieFN (Aspiring programmer)

**Comment by hothilux-21** [comment]
No additional req for this sort of idea so I'm first...

**Comment by TheCakeIsNaOH** [comment, marked as answer]
@StormieFN package descriptions and summaries are already available in the CLI if you run with `--verbose`, e.g. `choco search quickcpu --verbose`, and in the GUI if you double click the package (or right click and select details).

**Reply by pauby** [reply]
`choco info` can also be used.

**Comment by hothilux-21** [comment]
Oh MB

sorry for that i didnt read choco -? enough
